bootcamp/settings.py

```diff
@@ -114,7 +114,9 @@ def get_var(name, default):
     'raven.contrib.django.raven_compat',
 
     # Our INSTALLED_APPS
+    'backends',
```

Comment:
I'm not sure but it's installed in micromasters so I figured it needed to be here too